releasedir/provider.go

```diff
@@ -103,6 +103,8 @@ func (p Provider) newBlobstore(dirPath string) boshblob.DigestBlobstore {
 		blobstore = boshblob.NewLocalBlobstore(p.fs, p.uuidGen, options)
 	case "s3":
 		blobstore = NewS3Blobstore(p.fs, p.uuidGen, options)
```

Comment:
Ah, sorry I said `Close` and meant `Delete` for not being implemented in S3Blobstore. Given that that implements an interface, its hard to find usage of that method. Since the current `S3Blobstore` panics, swapping in the appropriate client call for the panic is probably not a breaking change.